domain/src/Services/TransactionDispatcher/TransactionDispatcher.php

```diff
@@ -11,7 +11,7 @@
 use Domain\Services\TransactionDispatcher\Handlers\TransferHandler;
 use Domain\ValueObjects\Transaction;
 
-final class TransactionDispatcher
+class TransactionDispatcher
```

Comment:
ℹ️ Mockery wouldn't mock it otherwise. I guess these classes should implement an interface instead, and I should mock the interface